client/components/tinymce/plugins/media/commands.js

```diff
@@ -0,0 +1,14 @@
+export function insertMediaAsLink( editor, media ) {
```

Comment:
Can you add the `/** @format */` tag to this file to [auto run the reformatter](https://github.com/Automattic/wp-calypso/blob/master/bin/reformat-files.js)?

Reply:
Thanks! You should be all set - during wp-calypso’s [pre-commit hook](https://github.com/Automattic/wp-calypso/blob/master/bin/pre-commit-hook.js#L56), we run prettier on any changed files that contain the marker.